loading-bay/camera_manager.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+import json
+import cv2
+from magic_numbers import *
+from typing import List
+
+
+class CameraManager:
+    CONFIG_FILE_PATH = "/boot/frc.json"
+
+    def __init__(self) -> None:
+        """Initialises a Camera Manager"""
+        from cscore import CameraServer
+
+        self.cs = CameraServer.getInstance()
+        self.camera_configs = self.read_config_JSON()
+
+        self.cameras = [
+            self.start_camera(camera_config) for camera_config in self.camera_configs
+        ]
+
+        self.sinks = [self.cs.getVideo(camera=camera) for camera in self.cameras]
+        self.source = self.cs.putVideo("Driver_Stream", FRAME_WIDTH, FRAME_HEIGHT)
+
+        self.frame = np.zeros(shape=(FRAME_HEIGHT, FRAME_WIDTH, 3), dtype=np.uint8)
```

Comment:
You should put a comment here about why width and height are reversed from the cs call, as it's unintuitive and looks at first glance like a bug.